composer.json

```diff
@@ -25,8 +31,9 @@
         "symfony/assetic-bundle"               : "~2.6",
         "symfony/monolog-bundle"               : "~2.7",
         "symfony/swiftmailer-bundle"           : "~2.3",
-        "symfony/symfony"                      : "~2.8",
-        "twig/extensions"                      : "~1.2"
+        "symfony/symfony"                      : "dev-action as 3.1.0",
+        "twig/extensions"                      : "~1.2",
+        "dunglas/action-bundle"                : "dev-master"
```

Comment:
well, having the branch alias already set would still allow to write this constraint as ``~1.0@dev``